app/models/forms/escort.rb

```diff
@@ -0,0 +1,7 @@
+module Forms
+  class Escort < Forms::Base
```

Comment:
The reason is because we are using the MpsFormBuilder for the text_area_without_label method as it comes handy in this case.